my_type=hypothesis
proj=yarn
u_test=org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
h_list[1]=yarn.http.policy@@@yarn:NodeManager@@@-3@@@HTTPS_ONLY@@@HTTP_ONLY
proj: yarn
u_test: org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
h_list: yarn.http.policy@@@yarn:NodeManager@@@-3@@@HTTPS_ONLY@@@HTTP_ONLY
updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v1v2 running time = 20480904417
---> v1v2 test failed.
proj: yarn
u_test: org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
h_list: yarn.http.policy@@@yarn:NodeManager@@@-3@@@HTTPS_ONLY@@@HTTP_ONLY
vv_mode: v1v2
result: FAIL
failureMessage: NMWebapps failed to start.
stackTrace: org.apache.hadoop.yarn.exceptions.YarnRuntimeException: NMWebapps failed to start.
	at org.apache.hadoop.yarn.server.nodemanager.webapp.WebServer.serviceStart(WebServer.java:117)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.service.CompositeService.serviceStart(CompositeService.java:121)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.yarn.server.MiniYARNCluster$NodeManagerWrapper.serviceStart(MiniYARNCluster.java:625)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.service.CompositeService.serviceStart(CompositeService.java:121)
	at org.apache.hadoop.yarn.server.MiniYARNCluster.serviceStart(MiniYARNCluster.java:322)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.yarn.client.api.impl.BaseAMRMClientTest.createClusterAndStartApplication(BaseAMRMClientTest.java:115)
	at org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints.testAMRMClientWithPlacementConstraintsByScheduler(TestAMRMClientPlacementConstraints.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.apache.hadoop.yarn.webapp.WebAppException: Error starting http server
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:443)
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:428)
	at org.apache.hadoop.yarn.server.nodemanager.webapp.WebServer.serviceStart(WebServer.java:112)
	... 22 more
Caused by: java.net.BindException: Port in use: 0.0.0.0:8042
	at org.apache.hadoop.http.HttpServer2.constructBindException(HttpServer2.java:1218)
	at org.apache.hadoop.http.HttpServer2.bindForSinglePort(HttpServer2.java:1240)
	at org.apache.hadoop.http.HttpServer2.openListeners(HttpServer2.java:1299)
	at org.apache.hadoop.http.HttpServer2.start(HttpServer2.java:1154)
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:439)
	... 24 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method)
	at sun.nio.ch.Net.bind(Net.java:433)
	at sun.nio.ch.Net.bind(Net.java:425)
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:220)
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:85)
	at org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:351)
	at org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:319)
	at org.apache.hadoop.http.HttpServer2.bindListener(HttpServer2.java:1205)
	at org.apache.hadoop.http.HttpServer2.bindForSinglePort(HttpServer2.java:1236)
	... 27 more


updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v1v1 running time = 34796825810
updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v2v2 running time = 32821810895
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v1v2 running time = 24289348850
---> v1v2 test failed.
proj: yarn
u_test: org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
h_list: yarn.http.policy@@@yarn:NodeManager@@@-3@@@HTTPS_ONLY@@@HTTP_ONLY
vv_mode: v1v2
result: FAIL
failureMessage: NMWebapps failed to start.
stackTrace: org.apache.hadoop.yarn.exceptions.YarnRuntimeException: NMWebapps failed to start.
	at org.apache.hadoop.yarn.server.nodemanager.webapp.WebServer.serviceStart(WebServer.java:117)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.service.CompositeService.serviceStart(CompositeService.java:121)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.yarn.server.MiniYARNCluster$NodeManagerWrapper.serviceStart(MiniYARNCluster.java:625)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.service.CompositeService.serviceStart(CompositeService.java:121)
	at org.apache.hadoop.yarn.server.MiniYARNCluster.serviceStart(MiniYARNCluster.java:322)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.yarn.client.api.impl.BaseAMRMClientTest.createClusterAndStartApplication(BaseAMRMClientTest.java:115)
	at org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints.testAMRMClientWithPlacementConstraintsByScheduler(TestAMRMClientPlacementConstraints.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.apache.hadoop.yarn.webapp.WebAppException: Error starting http server
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:443)
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:428)
	at org.apache.hadoop.yarn.server.nodemanager.webapp.WebServer.serviceStart(WebServer.java:112)
	... 22 more
Caused by: java.net.BindException: Port in use: 0.0.0.0:8042
	at org.apache.hadoop.http.HttpServer2.constructBindException(HttpServer2.java:1218)
	at org.apache.hadoop.http.HttpServer2.bindForSinglePort(HttpServer2.java:1240)
	at org.apache.hadoop.http.HttpServer2.openListeners(HttpServer2.java:1299)
	at org.apache.hadoop.http.HttpServer2.start(HttpServer2.java:1154)
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:439)
	... 24 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method)
	at sun.nio.ch.Net.bind(Net.java:433)
	at sun.nio.ch.Net.bind(Net.java:425)
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:220)
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:85)
	at org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:351)
	at org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:319)
	at org.apache.hadoop.http.HttpServer2.bindListener(HttpServer2.java:1205)
	at org.apache.hadoop.http.HttpServer2.bindForSinglePort(HttpServer2.java:1236)
	... 27 more


updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v1v1 running time = 34952033737
updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v2v2 running time = 31664746540
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v1v2 running time = 24511217136
---> v1v2 test failed.
proj: yarn
u_test: org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
h_list: yarn.http.policy@@@yarn:NodeManager@@@-3@@@HTTPS_ONLY@@@HTTP_ONLY
vv_mode: v1v2
result: FAIL
failureMessage: NMWebapps failed to start.
stackTrace: org.apache.hadoop.yarn.exceptions.YarnRuntimeException: NMWebapps failed to start.
	at org.apache.hadoop.yarn.server.nodemanager.webapp.WebServer.serviceStart(WebServer.java:117)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.service.CompositeService.serviceStart(CompositeService.java:121)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.yarn.server.MiniYARNCluster$NodeManagerWrapper.serviceStart(MiniYARNCluster.java:625)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.service.CompositeService.serviceStart(CompositeService.java:121)
	at org.apache.hadoop.yarn.server.MiniYARNCluster.serviceStart(MiniYARNCluster.java:322)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.yarn.client.api.impl.BaseAMRMClientTest.createClusterAndStartApplication(BaseAMRMClientTest.java:115)
	at org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints.testAMRMClientWithPlacementConstraintsByScheduler(TestAMRMClientPlacementConstraints.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.apache.hadoop.yarn.webapp.WebAppException: Error starting http server
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:443)
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:428)
	at org.apache.hadoop.yarn.server.nodemanager.webapp.WebServer.serviceStart(WebServer.java:112)
	... 22 more
Caused by: java.net.BindException: Port in use: 0.0.0.0:8042
	at org.apache.hadoop.http.HttpServer2.constructBindException(HttpServer2.java:1218)
	at org.apache.hadoop.http.HttpServer2.bindForSinglePort(HttpServer2.java:1240)
	at org.apache.hadoop.http.HttpServer2.openListeners(HttpServer2.java:1299)
	at org.apache.hadoop.http.HttpServer2.start(HttpServer2.java:1154)
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:439)
	... 24 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method)
	at sun.nio.ch.Net.bind(Net.java:433)
	at sun.nio.ch.Net.bind(Net.java:425)
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:220)
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:85)
	at org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:351)
	at org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:319)
	at org.apache.hadoop.http.HttpServer2.bindListener(HttpServer2.java:1205)
	at org.apache.hadoop.http.HttpServer2.bindForSinglePort(HttpServer2.java:1236)
	... 27 more


updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v1v1 running time = 36462381363
updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v2v2 running time = 40071618229
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v1v2 running time = 19542346456
---> v1v2 test failed.
proj: yarn
u_test: org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
h_list: yarn.http.policy@@@yarn:NodeManager@@@-3@@@HTTPS_ONLY@@@HTTP_ONLY
vv_mode: v1v2
result: FAIL
failureMessage: NMWebapps failed to start.
stackTrace: org.apache.hadoop.yarn.exceptions.YarnRuntimeException: NMWebapps failed to start.
	at org.apache.hadoop.yarn.server.nodemanager.webapp.WebServer.serviceStart(WebServer.java:117)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.service.CompositeService.serviceStart(CompositeService.java:121)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.yarn.server.MiniYARNCluster$NodeManagerWrapper.serviceStart(MiniYARNCluster.java:625)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.service.CompositeService.serviceStart(CompositeService.java:121)
	at org.apache.hadoop.yarn.server.MiniYARNCluster.serviceStart(MiniYARNCluster.java:322)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.yarn.client.api.impl.BaseAMRMClientTest.createClusterAndStartApplication(BaseAMRMClientTest.java:115)
	at org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints.testAMRMClientWithPlacementConstraintsByScheduler(TestAMRMClientPlacementConstraints.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.apache.hadoop.yarn.webapp.WebAppException: Error starting http server
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:443)
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:428)
	at org.apache.hadoop.yarn.server.nodemanager.webapp.WebServer.serviceStart(WebServer.java:112)
	... 22 more
Caused by: java.net.BindException: Port in use: 0.0.0.0:8042
	at org.apache.hadoop.http.HttpServer2.constructBindException(HttpServer2.java:1218)
	at org.apache.hadoop.http.HttpServer2.bindForSinglePort(HttpServer2.java:1240)
	at org.apache.hadoop.http.HttpServer2.openListeners(HttpServer2.java:1299)
	at org.apache.hadoop.http.HttpServer2.start(HttpServer2.java:1154)
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:439)
	... 24 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method)
	at sun.nio.ch.Net.bind(Net.java:433)
	at sun.nio.ch.Net.bind(Net.java:425)
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:220)
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:85)
	at org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:351)
	at org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:319)
	at org.apache.hadoop.http.HttpServer2.bindListener(HttpServer2.java:1205)
	at org.apache.hadoop.http.HttpServer2.bindForSinglePort(HttpServer2.java:1236)
	... 27 more


updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v1v1 running time = 28931506735
updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v2v2 running time = 35396113062
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v1v2 running time = 20328587551
---> v1v2 test failed.
proj: yarn
u_test: org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
h_list: yarn.http.policy@@@yarn:NodeManager@@@-3@@@HTTPS_ONLY@@@HTTP_ONLY
vv_mode: v1v2
result: FAIL
failureMessage: NMWebapps failed to start.
stackTrace: org.apache.hadoop.yarn.exceptions.YarnRuntimeException: NMWebapps failed to start.
	at org.apache.hadoop.yarn.server.nodemanager.webapp.WebServer.serviceStart(WebServer.java:117)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.service.CompositeService.serviceStart(CompositeService.java:121)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.yarn.server.MiniYARNCluster$NodeManagerWrapper.serviceStart(MiniYARNCluster.java:625)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.service.CompositeService.serviceStart(CompositeService.java:121)
	at org.apache.hadoop.yarn.server.MiniYARNCluster.serviceStart(MiniYARNCluster.java:322)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.yarn.client.api.impl.BaseAMRMClientTest.createClusterAndStartApplication(BaseAMRMClientTest.java:115)
	at org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints.testAMRMClientWithPlacementConstraintsByScheduler(TestAMRMClientPlacementConstraints.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.apache.hadoop.yarn.webapp.WebAppException: Error starting http server
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:443)
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:428)
	at org.apache.hadoop.yarn.server.nodemanager.webapp.WebServer.serviceStart(WebServer.java:112)
	... 22 more
Caused by: java.net.BindException: Port in use: 0.0.0.0:8042
	at org.apache.hadoop.http.HttpServer2.constructBindException(HttpServer2.java:1218)
	at org.apache.hadoop.http.HttpServer2.bindForSinglePort(HttpServer2.java:1240)
	at org.apache.hadoop.http.HttpServer2.openListeners(HttpServer2.java:1299)
	at org.apache.hadoop.http.HttpServer2.start(HttpServer2.java:1154)
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:439)
	... 24 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method)
	at sun.nio.ch.Net.bind(Net.java:433)
	at sun.nio.ch.Net.bind(Net.java:425)
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:220)
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:85)
	at org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:351)
	at org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:319)
	at org.apache.hadoop.http.HttpServer2.bindListener(HttpServer2.java:1205)
	at org.apache.hadoop.http.HttpServer2.bindForSinglePort(HttpServer2.java:1236)
	... 27 more


updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v1v1 running time = 28271472713
updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v2v2 running time = 31511403257
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v1v2 running time = 27467522558
---> v1v2 test failed.
proj: yarn
u_test: org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
h_list: yarn.http.policy@@@yarn:NodeManager@@@-3@@@HTTPS_ONLY@@@HTTP_ONLY
vv_mode: v1v2
result: FAIL
failureMessage: NMWebapps failed to start.
stackTrace: org.apache.hadoop.yarn.exceptions.YarnRuntimeException: NMWebapps failed to start.
	at org.apache.hadoop.yarn.server.nodemanager.webapp.WebServer.serviceStart(WebServer.java:117)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.service.CompositeService.serviceStart(CompositeService.java:121)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.yarn.server.MiniYARNCluster$NodeManagerWrapper.serviceStart(MiniYARNCluster.java:625)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.service.CompositeService.serviceStart(CompositeService.java:121)
	at org.apache.hadoop.yarn.server.MiniYARNCluster.serviceStart(MiniYARNCluster.java:322)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.yarn.client.api.impl.BaseAMRMClientTest.createClusterAndStartApplication(BaseAMRMClientTest.java:115)
	at org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints.testAMRMClientWithPlacementConstraintsByScheduler(TestAMRMClientPlacementConstraints.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.apache.hadoop.yarn.webapp.WebAppException: Error starting http server
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:443)
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:428)
	at org.apache.hadoop.yarn.server.nodemanager.webapp.WebServer.serviceStart(WebServer.java:112)
	... 22 more
Caused by: java.net.BindException: Port in use: 0.0.0.0:8042
	at org.apache.hadoop.http.HttpServer2.constructBindException(HttpServer2.java:1218)
	at org.apache.hadoop.http.HttpServer2.bindForSinglePort(HttpServer2.java:1240)
	at org.apache.hadoop.http.HttpServer2.openListeners(HttpServer2.java:1299)
	at org.apache.hadoop.http.HttpServer2.start(HttpServer2.java:1154)
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:439)
	... 24 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method)
	at sun.nio.ch.Net.bind(Net.java:433)
	at sun.nio.ch.Net.bind(Net.java:425)
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:220)
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:85)
	at org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:351)
	at org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:319)
	at org.apache.hadoop.http.HttpServer2.bindListener(HttpServer2.java:1205)
	at org.apache.hadoop.http.HttpServer2.bindForSinglePort(HttpServer2.java:1236)
	... 27 more


updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v1v1 running time = 36741354926
updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v2v2 running time = 34360602141
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v1v2 running time = 23221739695
---> v1v2 test failed.
proj: yarn
u_test: org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
h_list: yarn.http.policy@@@yarn:NodeManager@@@-3@@@HTTPS_ONLY@@@HTTP_ONLY
vv_mode: v1v2
result: FAIL
failureMessage: NMWebapps failed to start.
stackTrace: org.apache.hadoop.yarn.exceptions.YarnRuntimeException: NMWebapps failed to start.
	at org.apache.hadoop.yarn.server.nodemanager.webapp.WebServer.serviceStart(WebServer.java:117)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.service.CompositeService.serviceStart(CompositeService.java:121)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.yarn.server.MiniYARNCluster$NodeManagerWrapper.serviceStart(MiniYARNCluster.java:625)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.service.CompositeService.serviceStart(CompositeService.java:121)
	at org.apache.hadoop.yarn.server.MiniYARNCluster.serviceStart(MiniYARNCluster.java:322)
	at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
	at org.apache.hadoop.yarn.client.api.impl.BaseAMRMClientTest.createClusterAndStartApplication(BaseAMRMClientTest.java:115)
	at org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints.testAMRMClientWithPlacementConstraintsByScheduler(TestAMRMClientPlacementConstraints.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.apache.hadoop.yarn.webapp.WebAppException: Error starting http server
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:443)
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:428)
	at org.apache.hadoop.yarn.server.nodemanager.webapp.WebServer.serviceStart(WebServer.java:112)
	... 22 more
Caused by: java.net.BindException: Port in use: 0.0.0.0:8042
	at org.apache.hadoop.http.HttpServer2.constructBindException(HttpServer2.java:1218)
	at org.apache.hadoop.http.HttpServer2.bindForSinglePort(HttpServer2.java:1240)
	at org.apache.hadoop.http.HttpServer2.openListeners(HttpServer2.java:1299)
	at org.apache.hadoop.http.HttpServer2.start(HttpServer2.java:1154)
	at org.apache.hadoop.yarn.webapp.WebApps$Builder.start(WebApps.java:439)
	... 24 more
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method)
	at sun.nio.ch.Net.bind(Net.java:433)
	at sun.nio.ch.Net.bind(Net.java:425)
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:220)
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:85)
	at org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:351)
	at org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:319)
	at org.apache.hadoop.http.HttpServer2.bindListener(HttpServer2.java:1205)
	at org.apache.hadoop.http.HttpServer2.bindForSinglePort(HttpServer2.java:1236)
	... 27 more


updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v1v1 running time = 39913864327
updating test result for file org.apache.hadoop.yarn.client.api.impl.TestAMRMClientPlacementConstraints#testAMRMClientWithPlacementConstraintsByScheduler
v2v2 running time = 33974514950
---> v1v1 and v2v2 test test suceeded.
early stop after 7 is satisfied
v1v2 failed with probability 7 out of 7
v1v1v2v2 failed with probability 0 out of 7
Total execution time in seconds : 639
0
